readme.txt

```diff
@@ -4,6 +4,7 @@
 
 = [4.12.10] TBD =
 
+* Tweak - Introduce the `tribe_doing_shortcode()` template tag to check if one of our shortcodes is being done. [ET-904]
```

Comment:
Should we limit to just our shortcodes?

Reply:
It's limited to our shortcodes. We do the early bail if `! $this->is_shortcode_registered( $tag )`